tests/ci/run_benchmark_build_tests.sh

```diff
@@ -4,61 +4,78 @@
 
 source tests/ci/common_posix_setup.sh
 
-install_dir="$(pwd)/../libcrypto_install_dir"
+scratch_folder=${SYS_ROOT}/"benchmark-scratch"
+install_dir="${scratch_folder}/libcrypto_install_dir"
 openssl_url='https://github.com/openssl/openssl.git'
 openssl_1_1_branch='OpenSSL_1_1_1-stable'
 openssl_1_0_branch='OpenSSL_1_0_2-stable'
-# OpenSSL 3.0 branch still isn't stable yet, so we lock down
-# a tag to prevent build failures that they introduced from
-# failing our CI.
-openssl_3_0_branch='openssl-3.0.5'
+openssl_3_0_branch='openssl-3.1'
```

Comment:
Variable named `3.0` but really `3.1`.

Does this reflect when capturing and graphing later on in the workflow?

Reply:
Yeah, it uses the name of the branch in the metric. I can fix this and add a 3.2 test.